src/templates/plugin.js

```diff
@@ -27,11 +28,12 @@ export default ({app, store}) => {
   })
   app.i18n = new VueI18n({
     locale: store.state['i18n'].language,
-    fallbackLocale: options.languages[0],
+    fallbackLocale: options.defaultLanguage || options.languages[0],
     messages: messages,
     silentTranslationWarn: true
   })
 
+  Vue.availableLanguages = Object.keys(messages)
```

Comment:
What's the reason to get the *available languages* from the messages/phrases, and not from the module config (`options.languages`)? I think it'd be good to do it consistently across the module, and the latter seems a more robust source for that, wdyt?